libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/SmartStore.java

```diff
@@ -128,7 +128,7 @@
     public static synchronized void changeKey(SQLiteDatabase db, String oldKey, String newKey) {
     	synchronized(db) {
 	        if (newKey != null && !newKey.trim().equals("")) {
-	            db.execSQL("PRAGMA rekey = '" + newKey + "'");
+	            db.query("PRAGMA rekey = '" + newKey + "'");
```

Comment:
Getting an error when calling with db.execSQL (in sqlcipher 4.3.0, pragma returns ok).